File: ClientApp/src/app/_models/image.ts
export class Image{

  id:Number;
  name:string;
  description:string;
  dateAdded:Date;
  isProfile:boolean;

  constructor(id:number,name:string,description:string,dateAdded:Date,isProfile:boolean) {
     this.id=id;
     this.name=name;
     this.description=description;
     this.dateAdded=dateAdded;
     this.isProfile=isProfile;
  }

}

File: ServerApp/SocialApp.EntityLayer/Concrete/Image.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SocialApp.EntityLayer.Concrete
{
    public class Image
    {
        public int Id { get; set; }
        public string Name { get; set; }

        public string Description { get; set; } 
        public DateTime DateAdded { get; set; }

        public bool IsProfile { get; set; }

        public int? UserId { get; set; }
        public User? User { get; set; }
    }
}
